src/esp/gfx_batch/Renderer.cpp

```diff
@@ -1206,7 +1206,9 @@ void Renderer::draw(Mn::GL::AbstractFramebuffer& framebuffer) {
           .drawsSorted[i]
           /* Extract normal matrix */
           .setNormalMatrix(state_->absoluteTransformationsSorted[i]
-                               .transformationMatrix.normalMatrix())
+                               .transformationMatrix.rotationScaling()
+                               .inverted()
+                               .transposed())
```

Comment:
Actually, can you revert this part in the batch renderer? 

I decided to keep the `normalMatrix()` in (and just fix it) because it's a lot more convenient than having to remember and type out `foo.rotationScaling().inverted().transposed()`. So, once I get to doing the rest of the fix, I would be changing this back to `normalMatrix()` anyway.

The remaining changes are extracting the determinant calculation so they can't use `normalMatrix()` anymore.

Reply:
Ok, I'll take care of that on Monday :)